tests/functional/pod_and_daemons/test_odf_resources_requests_and_limits.py

```
@@ -54,39 +62,95 @@ def test_live_resources_presence_and_format(self):
+            logger.assertion(
+                f"Resource structural compliance format check: result={validation['result']}"
+            )
+            assert validation["result"], error_message
+            logger.info(
+                "Success! All live rescheduled pod resource metrics exist and are well-formed."
+            )
+
+        except Exception as e:
+            # Guideline: Use logger.exception() within except blocks to automatically capture tracebacks at ERROR level
+            logger.exception(
+                f"An unexpected failure occurred during the live resources evaluation: {e}"
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)